Feature/get user by id

Kiresorg opened the pull request:
Please review. Closes #2

m-labbe commented on routes/users.js:
```diff
@@ -10,5 +10,15 @@ router.get("/users", async (req, res) => {
 
+router.get("/users/:id", async (req, res) => 
+{
+  const userid = parseInt(req.params.id);
```
Can we use camelCase for this variable name?

Kiresorg replied:
Done

m-labbe commented on routes/users.js:
```diff
@@ -10,5 +10,15 @@ router.get("/users", async (req, res) => {
+  {
+    res.send({"error": "Specified User not found"});
+  }
+  var user = users.find(item => item.id === userid);
```
Let's make this a `const`

Kiresorg replied:
Done

m-labbe reviewed (changes requested):
Look good! Can you add some tests for this endpoint?

Kiresorg commented:
Made those changes and added tests. Great feedback. I also noticed I was gong to merge into main which would be bad :) I changed it to release branch.

m-labbe reviewed (approved):
LGTM!
